src/sentry/api/endpoints/project_backfill_similar_issues_embeddings_records.py

```diff
@@ -44,10 +46,18 @@ def post(self, request: Request, project) -> Response:
         if request.data.get("enable_ingestion"):
             enable_ingestion = request.data["enable_ingestion"] == "true"
 
+        if request.data.get("skip_processed_projects"):
+            skip_processed_projects = request.data["skip_processed_projects"] == "true"
```

Comment:
why use the string `"true"` instead of the boolean `True`?

Reply:
good catch, thanks!